src/AppBundle/Resources/config/routing.yml

```diff
@@ -0,0 +1,7 @@
+app_add_component:
+    path:     /add_component
```

Comment:
if you can, update the routing like `/components/add` and `/components/{componentId}/edit` (the last one will be useful after)